CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ All notable, unreleased changes to this project will be documented in this file.
   - Add new fields: `Product.attribute`, `Product.variant`
   - Add `sortBy` argument to `Product.media`
 - Allow assigning attribute value using its ID. Add to `AttributeValueInput` dedicated field for each input type. #11206 by @zedzior
+- Add ability to filter and sort products of a category - #10917 by @yemeksepeti-cihankarluk, @ogunheper
+  - Add `filter` argument to `Category.products`
+  - Add `sortBy` argument to `Category.products`
```

Comment:
Can you rebase your PR and move these changes to 3.10 section? Other than that I think this PR should be ready to merge.

Reply:
Rebased and moved under 3.10 @maarcingebala